https://bugs.gentoo.org/64971
skey(1) and skeyinit(1) describe md4 as the default hash algorithm,
which is no longer the case.

--- skey-1.1.5-orig/skey.1
+++ skey-1.1.5/skey.1
@@ -27,7 +27,7 @@
 .Pp
 .Em S/Key
 uses 64 bits of information, transformed by the
-.Tn MD4
+.Tn MD5
 algorithm into 6 English words.
 The user supplies the words to authenticate himself to programs like
 .Xr login 1
--- skey-1.1.5-orig/skeyinit.1	2012-01-04 20:24:22.000000000 +0100
+++ skey-1.1.5/skeyinit.1	2012-01-04 20:31:13.000000000 +0100
@@ -47,7 +47,7 @@
 (default is 100).
 .It Fl t Ar hash
 Selects the hash algorithm to use.
-Available choices are md4 (the default), md5 or sha1.
+Available choices are md4, md5 (the default) or sha1.
 .It Ar user
 The username to be changed/added.
 By default the current user is operated on, only root may
